css-build

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
Brunch is included in the repo so this shouldn't be necessary.

Reply:
Removed.